assets/scss/_variables_project.scss

```diff
@@ -7,6 +7,7 @@ $otel-colors: (
   'blue-light': #cee1ff,
   'purple': #425cc7,
   'light-green': #c0f587,
+  'dark-background': #262d2c,
```

Comment:
what's the purpose of this indirection? Is this used anywhere else except in line 59 in `_registry.scss`?

Reply:
The variable is supposed to add a darker background color in dark mode. It's used on line 59 but has been re-used on line 69 on the _registry.scss file. I'll customize the code to avoid repetition.